A/Antic/build_tarballs.jl

```diff
@@ -0,0 +1,47 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Antic"
+version = v"0.0.1+0"
```

Comment:
You can't specify a build version:
```suggestion
version = v"0.0.1"
```
Is really this the upstream version number?

Reply:
What Mose was saying is the the version here should generally be the same as the underlying library. It looks like the latest release of antic is 0.2, but you have built the master as it stands today. I'd suggest calling it 0.2.0, and then updating it as new releases are tagged in antic.

Reply:
Thanks, I have updated the version number.